_shared/pyproject.toml

```diff
@@ -0,0 +1,12 @@
+[project]
+name = "shared_utils"
+description = "Shared utilities for Velocitas toolchain components."
```

Comment:
What about putting these utilities inside velocitas_lib, rather than opening up another shared functions inside this repo?